nlpnet/attributes.py

```diff
@@ -86,7 +86,7 @@ def load_affixes(cls, codes, filename):
         try:
             with open(filename, 'rb') as f:
                 for line in f:
-                    affix = unicode(line.strip(), 'utf-8')
+                    affix = str(line.strip(), 'utf-8')
```

Comment:
I think `str` return bystrings in python 2 but unicode strings in python 3, I wonder whether the use of `six` for this kind of conversion wouldn't be better.